src/components/PlayerInfo.vue

```diff
@@ -90,30 +87,18 @@
                     <img class="readyButton" src="@/assets/readyButtonBig.png"
                          v-on:click="confirmReady">
                 </div>
-                <LeaveGame/>
             </div>
         </div>
-
-        <div class="ready" v-if="ready">
-            Waiting on others to ready up...
-            <LeaveGame class="leaveGameWaiting"/>
-        </div>
     </div>
 </template>
 
 <script>
-    import WebsocketService from "../services/WebsocketService";
-    import LeaveGame from "./LeaveGame"
-
     export default {
         name: 'PlayerInfo',
-        components: {
-            LeaveGame
-        },
+        components: {},
         methods: {
             confirmReady: function () {
-                const confirmReadyObj = {event: 'PlayerReady'};
-                WebsocketService.sendObj(this.$socket, confirmReadyObj);
+                this.$store.dispatch("stepTwoToQuestPhase");
             }
         },
         computed: {
```

Comment:
You can get ride of the computed object, ready isn't doing anything anymore.

Reply:
While you're at it, you can get rid of the playerInfo (and its ready property) object in the store altogether!  It won't be used after this.

Reply:
There's some other things in the `computed` object that we'll need to keep but I got rid of everything else you mentioned now!

Reply:
ahh, didn't see that github was cutting out some lines.  mb